service/arm_templates/virtual_network_setup.json

```diff
@@ -86,6 +121,18 @@
     "workerSubnetID": {
       "type": "string",
       "value": "[variables('workerSubnetID')]"
-    }
+    },
+    "apiLoadBalancerSubnetID": {
+      "type": "string",
+      "value": "[variables('apiLoadBalancerSubnetID')]"
+    },
+    "etcdLoadBalancerSubnetID": {
+      "type": "string",
+      "value": "[variables('etcdLoadBalancerSubnetID')]"
+    },
+    "ingressLoadBalancerSubnetID": {
+      "type": "string",
+      "value": "[variables('ingressLoadBalancerSubnetID')]"
+    },
   }
```

Comment:
I wonder if we need those outputs at all. Do we?

Reply:
Will remove that too :)